src/java.desktop/share/classes/javax/swing/plaf/metal/MetalTheme.java

```diff
@@ -65,7 +65,7 @@
  */
 public abstract class MetalTheme {
 
-    // Contants identifying the various Fonts that are Theme can support
+    // Constants identifying the various Fonts that a Theme can support
```

Comment:
I think this is possibly meant to say "constants that *our* theme can support"?

Reply:
Please submit a new JBS issue for it if you can; if you can't, I can submit one on your behalf.

Possibly it meant to say “_the_ theme”.

Reply:
I think the text is correct here: …that _**a** Theme_ can support

`MetalTheme` is a superclass for classes which implement Themes. Thus the constants define fonts which _a particular theme_ may use.

Reply:
Makes sense.